### ISSUE TITLE
Suggestion - Wilks to Age Adjust for Masters

### ISSUE THREAD

**Issue by joezap** [sstangl/openpowerlifting#724]
Hello,

It would be nice to have the Wilks calculated with the age adjust for masters lifters. For example the USPA uses the McCulloch Masters Formula (revised) when determining standing places and best lifters.

To me it seem masters are not fairly represented in the ranking on the Open Powerlifting website.

Thanks for considering this change,
Joe

**Comment by sstangl** [comment]
Hey Joe, thanks for the report. We are going to do this. We do actually calculate McCulloch/Foster coefficients at the moment, but don't have a way to show it on the site yet. We're rewriting things at the moment, and age coefficients will be part of that rewrite.

Age adjustment depends on #57 for inexact age adjustment.

**Comment by sstangl** [comment]
Just for clarity, we are going to have there be two options -- `Wilks` and `Age-Adjusted Wilks` (in addition to other options, such as `Glossbrenner`, `Allometric Scaling`, etc.).

The default option will remain `Wilks`.

**Comment by joezap** [comment]
Thanks Sean! Keep up the great work. I remember when you started this project and I'm glad it has been a success. The USPA website even links to it with the USPA filtered already.
